demos/Location/main.py

```diff
@@ -0,0 +1,126 @@
+import gi
+
+gi.require_version("Xdp", "1.0")
+gi.require_version("XdpGtk4", "1.0")
+from gi.repository import Gio, Xdp, XdpGtk4, GLib
+import workbench
+
+portal = Xdp.Portal()
+parent = XdpGtk4.parent_new_gtk(workbench.window)
+
+revealer = workbench.builder.get_object("revealer")
+start = workbench.builder.get_object("start")
+close = workbench.builder.get_object("close")
+distance_threshold = workbench.builder.get_object("distance_threshold")
+time_threshold = workbench.builder.get_object("time_threshold")
+accuracy_button = workbench.builder.get_object("accuracy_button")
+
+latitude_label = workbench.builder.get_object("latitude")
+longitude_label = workbench.builder.get_object("longitude")
+accuracy_label = workbench.builder.get_object("accuracy")
+altitude_label = workbench.builder.get_object("altitude")
+speed_label = workbench.builder.get_object("speed")
+heading_label = workbench.builder.get_object("heading")
+description_label = workbench.builder.get_object("description")
+timestamp_label = workbench.builder.get_object("timestamp")
+
+locationAccuracy = Xdp.LocationAccuracy.EXACT
```

Comment:
Please name all variables in `snake_case`. 

I guess it's the same in the JS demo. Also it's really not great that `time_threshold` and `timeThreshold` exist as variables (again same in JS). Can you change that also in the JS demo? Thanks a lot!